Assignment4/Josesilva99-z170360/Task06.ipynb

```diff
@@ -0,0 +1,339 @@
+        "id": "-Q_JjPKdhrZN",
+        "outputId": "41a5beb5-f320-48dd-a6bb-2c88e93fd237",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        }
+      },
+      "source": [
+        "g.add((ns.JaneSmith, RDF.type, ns.Researcher))\n",
+        "#Comprobamos:\n",
+        "for s, p, o in g.triples((ns.JaneSmith, RDF.type, ns.Researcher)):\n",
+        "  print(s,p,o)"
+      ],
+      "execution_count": 37,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "http://somewhere#JaneSmith http://www.w3.org/1999/02/22-rdf-syntax-ns#type http://somewhere#Researcher\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tD383J__eHfV"
+      },
+      "source": [
+        "**TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "I61KSuxMiLNo",
+        "outputId": "fac2fa1c-8e5f-42ff-b9a3-5316950b94c2",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 84
+        }
+      },
+      "source": [
+        "VCARD = Namespace(\"http://www.w3.org/2001/vcard-rdf/3.0#\")\n",
+        "g.add((ns.JaneSmith, VCARD.FN, Literal(\"Jane Smith\")))\n",
+        "g.add((ns.JaneSmith, VCARD.Given, Literal(\"Jane\")))\n",
+        "g.add((ns.JaneSmith, VCARD.Family, Literal(\"Smith\")))\n",
+        "#Comprobamos:\n",
+        "for s, p, o in g.triples((ns.JaneSmith, None, None)):\n",
+        "  print(s,p,o)"
+      ],
+      "execution_count": 38,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "http://somewhere#JaneSmith http://www.w3.org/2001/vcard-rdf/3.0#Family Smith\n",
+            "http://somewhere#JaneSmith http://www.w3.org/2001/vcard-rdf/3.0#FN Jane Smith\n",
+            "http://somewhere#JaneSmith http://www.w3.org/1999/02/22-rdf-syntax-ns#type http://somewhere#Researcher\n",
+            "http://somewhere#JaneSmith http://www.w3.org/2001/vcard-rdf/3.0#Given Jane\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "GyZOMndoeUj4"
+      },
+      "source": [
+        "**TASK 6.5: Add UPM as the university where John Smith works**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "XZXZFerikBcZ",
+        "outputId": "ffd643f7-e733-4e6f-ac74-3f95b28150da",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 50
+        }
+      },
+      "source": [
+        "g.add((ns.UPM, RDF.type, ns.University))\n",
+        "#Comprobamos:\n",
+        "for s, p, o in g.triples((ns.UPM, RDF.type, ns.University)):\n",
+        "  print(s,p,o)\n",
+        "g.add((ns.JohnSmith, VCARD.Work, ns.UPM))\n",
```

Comment:
this property does not exist in VCARD, it is okay to look what your colleagues are doing, but not all will be correct ;-). Please change it by a correct property

Reply:
Now is it correct? The new property I have taken from here [https://www.w3.org/TR/vcard-rdf/]